content/css/concepts/inheritance/inheritance.md

```diff
@@ -0,0 +1,62 @@
+---
+Title: 'Inheritance'
+Description: 'Inheritance describes how certain styles are initialized or computed depending on the CSS property and whether a value was set.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Inheritance'
+  - 'Style'
+CatalogContent:
+  - 'learn-css'
+  - 'paths/front-end-engineer-career-path'
+---
+
+CSS inheritance describes how certain styles are initialized or computed depending on the CSS property and whether a value was set. Some properties are inherited with an initial, default value. Other properties are non-inherited and set to the computed value of its containing element by default.
+
+## Inherited properties
+
+Inherited properties apply to the element and its children.
+
+| Property                                                                                    | Description                                                                                  |
+| ------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------- |
+| [`color`](https://www.codecademy.com/resources/docs/css/colors/color)                       | Sets the text color of an element.                                                           |
+| [`visibility`](https://www.codecademy.com/resources/docs/css/visibility)                    | Hides or reveals an element without affecting its placement within the page.                 |
```

Comment:
Super nit-picky suggestion but could we make sure the rows in these tables are alphabetized by "Property", such as moving this row for `visibility` down below? (between `text-transform` and `word-spacing`)